probe-rs/src/rtt.rs

```diff
@@ -260,7 +411,7 @@ impl Rtt {
             .into_iter()
             .filter_map(|range| {
                 let range_len = match range.end.checked_sub(range.start) {
-                    Some(v) if v < Self::MIN_SIZE as u64 => return None,
+                    Some(v) if v < (if is_64_bit {std::mem::size_of::<RttControlBlockHeaderInner<u64>>()} else {std::mem::size_of::<RttControlBlockHeaderInner<u32>>()} as u64) => return None,
```

Comment:
This isn't the most elegant way, 

- Complex guard conditions like this are hard to understand.
- This is the third place where you implement the `bool -> pointer width` conversion. You should probably add a `RttControlBlockHeader(Inner?)::size(is_64_bit: bool)` function.

Reply:
Thank you for your review!
I fixed it 22e9c7c66e0aee84ab7c828710b6a073ed78cf54